config/locales/pt-BR/views.yml

```diff
@@ -27,7 +27,7 @@ pt-BR:
         Pessoal,
       </p>
       <p>
-        Preencham o punch entre os dias 16 e 15.
+        Preencham o punch entre os dias 15 e 16.
```

Comment:
I believe we can improve this message adding months to it, something like: `Preencham o punch entre os dias 15/<last month> e 16/<current month>`.